docs/genai/howto/install.md

```diff
@@ -23,61 +29,149 @@ The CUDA toolkit can be downloaded from the [CUDA Toolkit Archive](https://devel
-
 #### CUDA 11
 
 ```bash
-pip install numpy
-pip install onnxruntime-genai-cuda --pre --index-url=https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-genai/pypi/simple/
+# Install the nightly version of ONNX Runtime. Update this to the released version when it is available.
+pip install ort-nightly-gpu --extra-index-url
+https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ort-cuda-11-nightly/pypi/simple
+
+
+# Install onnxruntime-genai built for CUDA 11
+pip install onnxruntime-genai-cuda-11 --pre
 ```
 
 #### CUDA 12
 
 ```bash
-pip install numpy
-pip install onnxruntime-genai-cuda --pre --index-url=https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-cuda-12/pypi/simple/
+# Install ORT nightly CUDA 12, change this to released version when it is released
+pip install onnxruntime-gpu --extra-index-url https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-cuda-12/pypi/simple/
+
+# Install onnxruntime-genai build for CUDA 12
+pip install onnxruntime-genai-cuda --pre
 ```
 
 ## Nuget packages
 
-Note: only one of these packages should be installed in your application.
+Note: only one of these set of packages (CPU, DirectML, CUDA) should be installed in your project.
+
+### CPU
+
+Add the following lines to a `nuget.config` file in the same folder as your `.csproj` file.
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <clear />
+    <add key="ORT-Nightly" value="https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/nuget/v3/index.json" />
+  </packageSources>
+</configuration>
+```
 
 ```bash
+dotnet add package Microsoft.ML.OnnxRuntime --source ORT-Nightly --prerelease
 dotnet add package Microsoft.ML.OnnxRuntimeGenAI --prerelease
 ```
 
-For the package that has been optimized for CUDA:
+### CUDA 
+
+#### CUDA 11
+
+Add the following lines to a `nuget.config` file in the same folder as your `.csproj` file.
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <clear />
+    <add key="ORT-Nightly" value="https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/nuget/v3/index.json" />
+  </packageSources>
+</configuration>
+```
 
 ```bash
+dotnet add package Microsoft.ML.OnnxRuntime.Gpu --source ORT-Nightly --prerelease
+dotnet add package Microsoft.ML.OnnxRuntimeGenAI.Cuda --source --prerelease
+```
+
+#### CUDA 12
+
+Add the following lines to a `nuget.config` file in the same folder as your `.csproj` file.
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <clear />
+    <add key="ORT-Nightly" value="https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/nuget/v3/index.json" />
+  </packageSources>
+</configuration>
+```
+
+
+```bash
+dotnet add package Microsoft.ML.OnnxRuntime.Gpu --source ORT-Nightly --prerelease
```

Comment:
We do not need this. The ort-genai will automatically use the source and pull the version for the users.

Reply:
Fixed